packages/design-system-docs/src/pages/components/Dropdown/Dropdown.example.jsx

```diff
@@ -67,6 +68,18 @@ ReactDOM.render(
         <option value="2-3">Option 6</option>
       </optgroup>
     </Dropdown>
+    <div className="example--wrapper example--inverse">
+      <Dropdown
+        labelClassName="ds-u-margin-top--0"
+        options={dropdownOptions}
+        defaultValue="1"
+        errorMessage="Example error message"
+        hint="Helpful hint text"
+        label="Inverse dropdown"
```

Comment:
On other inverse versions, we use the language `Inverse example` - We might want to be consistent here for the label and use `Inverse example`

Reply:
I'm wondering if we should use this type of format for all inverse examples `Inverse COMPONENT-NAME example`

Reply:
Yeah I noticed that as well, the reason why I use "Inverse dropdown" here is because the other dropdown examples use "dropdown" in the labels. The components that use "Inverse example" dont mention the component name in it. Maybe it would be better to avoid using dropdown in the label name?